diff -ur kdegames-3.5.10/libksirtet/base/Makefile.am kdegames-3.5.10-ftbfs/libksirtet/base/Makefile.am
--- kdegames-3.5.10/libksirtet/base/Makefile.am	2005-10-10 16:56:23.000000000 +0200
+++ kdegames-3.5.10-ftbfs/libksirtet/base/Makefile.am	2010-02-15 17:05:54.000000000 +0100
@@ -7,6 +7,7 @@
 
 noinst_LTLIBRARIES        = libksirtetbase.la
 libksirtetbase_la_LDFLAGS = $(all_libraries)
+libksirtetbase_la_LIBADD  = -lkio
 
 noinst_HEADERS            = kzoommainwindow.h piece.h gtetris.h factory.h highscores.h \
                             board.h settings.h field.h inter.h main.h
